mapper/orm-batch-jsr352/core/pom.xml

```diff
@@ -49,6 +49,33 @@
             <artifactId>jakarta.batch-api</artifactId>
         </dependency>
 
+        <!-- This dependency is only here to solve Moditect errors.
+             For some reason, the jakarta.transaction module requires jakarta.interceptor,
+             but the jakarta.transaction-api Maven module doesn't depend on jakarta.interceptor-api,
+             and that doesn't play well with Moditect, which ends up throwing an exception:
+               Execution add-module-infos of goal org.moditect:moditect-maven-plugin:1.0.0.RC2:add-module-info failed: Module jakarta.interceptor not found, required by jakarta.transaction
+         -->
+        <dependency>
+            <groupId>jakarta.interceptor</groupId>
+            <artifactId>jakarta.interceptor-api</artifactId>
+            <!-- DO NOT REMOVE and DO NOT MANAGE the version of this dependency. See the version property declaration. -->
+            <version>${version.jakarta.interceptor-api}</version>
+            <scope>provided</scope>
+        </dependency>
+        <!-- This dependency is only here to solve Moditect errors.
+             For some reason, the jakarta.transaction module requires jakarta.cdi,
+             but the jakarta.transaction-api Maven module doesn't depend on jakarta.enterprise.cdi-api,
+             and that doesn't play well with Moditect, which ends up throwing an exception:
+               Execution add-module-infos of goal org.moditect:moditect-maven-plugin:1.0.0.RC2:add-module-info failed: Module jakarta.interceptor not found, required by jakarta.transaction
+         -->
+        <dependency>
+            <groupId>jakarta.enterprise</groupId>
+            <artifactId>jakarta.enterprise.cdi-api</artifactId>
+            <!-- DO NOT REMOVE and DO NOT MANAGE the version of this dependency. See the version property declaration. -->
+            <version>${version.jakarta.enterprise}</version>
+            <scope>provided</scope>
+        </dependency>
+
```

Comment:
Did you check whether this workaround is still necessary? We've upgraded these dependencies multiple times since the workaround was first introduced.